packages/kotti-ui/source/kotti-popover/KtPopover.vue

```diff
@@ -12,6 +12,7 @@
 				<IconTextItem
 					v-for="(option, index) in options"
 					:key="index"
+					:data-test="option.dataTest"
```

Comment:
does `IconTextItem` accept the data-test 

what do u do with it, this doesn't eventually achieve ur goal
b/c u pass it down but you don't eventualy bind it on a lower level

Reply:
NVM - attributes are by default auto intehrited, unless a flag specifies otherwise, so I'll approve :)

Reply:
While the reasoning may not be correct, I still agree with what you’re saying though. Eventually, we may accidentally break this feature if it’s handled from only the parent’s side. I’ll add `dataTest` to the props